pkg/destroy/azure/azure.go

```diff
@@ -134,6 +136,8 @@ func New(logger logrus.FieldLogger, metadata *types.ClusterMetadata) (providers.
 		Logger:                      logger,
 		BaseDomainResourceGroupName: metadata.Azure.BaseDomainResourceGroupName,
 		CloudName:                   cloudName,
+		ZoneName:                    metadata.Azure.BaseDomainName,
+		APIURL:                      fmt.Sprintf("api.%s", metadata.ClusterName),
```

Comment:
not sure if it makes sense to persist APIURL state here, especially if we're also going to check for *.apps

Reply:
Wrong commit sorry. Fixed it